benchmark/index.js

```diff
@@ -25,69 +28,79 @@ const obj = {
       c: 's'
     }
   }
-}
+});
```

Comment:
stray ;

Reply:
more stray semi colons below, can you remove all (you can use `standard --fix` for convenience)